README.md

```diff
@@ -1 +1,272 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+    <img src="https://github.com/didierganthier/hello-microverse-cr-review/blob/master/murple_logo.png" alt="logo" width="150px" />
+  <br/>
```

Comment:
[OPTIONAL] Remove Microverse Logo. It was supposed to be part of the template. Now you have to customize it for your project.